src/zenml/integrations/gcp/vertex_custom_job_parameters.py

```diff
@@ -48,3 +61,4 @@ class VertexCustomJobParameters(BaseModel):
     boot_disk_type: str = "pd-ssd"
     persistent_resource_id: Optional[str] = None
     service_account: Optional[str] = None
+    advanced_training_job_args: Dict[str, Any] = {}
```

Comment:
The name feels weird to me, `advanced_training_job_args` inside the `CustomJobParameters`. Maybe just `additional_args`?